benchmark/metrics_parser.py

```diff
@@ -0,0 +1,276 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+import argparse
+from collections import defaultdict
+import csv
+import itertools
+import json
+
+
+WALL_CLOCK_KEY = u'-- WALL --'
+
+
+HEADERS = [
+    u'Flocker Version',
+    u'Nodes',
+    u'Containers',
+    u'Scenario',
+    u'Control Service CPU',
+    u'Dataset Agent CPU',
+    u'Container Agent CPU',
+    u'Containers Converged Within Limit',
+    u'Scenario Requests Within Limit',
+]
+
+
+def write_csv(results, filename):
+    with open(filename, 'w') as csvfile:
+        writer = csv.DictWriter(csvfile, fieldnames=HEADERS)
+        writer.writeheader()
+        writer.writerows(results)
+
+
+def mean(values):
+    if len(values) > 0:
+        return sum(values) / len(values)
```

Comment:
Convert the sum of values to float to ensure that this function returns a float, and not a truncated int.